ControlFlow/Program.cs

```diff
@@ -15,22 +15,48 @@ public static class ControlFlow
         // be validated).  Negative numbers should return "Invalid".
         public static string AreYouValid(int number)
         {
-            return default;
+            if (number > 1 & number < 10)
+            {
+                return "Valid";
+            }
+            else
+            {
+                return "Invalid";
+            }
+
+
         }
 
         // 1.2 - Write a method which takes two integers and returns the maximum of the two. If they
         // are the same, return either one.
         public static int Maximum(int number1, int number2)
         {
-            return default;
+            if(number1 >= number2)
+            {
+                return number1;
+            }    
+            else
+            {
+                return number2;
+            }
         }
 
         // 1.3 - Write a method which takes two integers, the width and height of an image. Then
         // return a string to tell if the image is "Landscape" or "Portrait". Return "Square" if
         // they are equal. Assume non-negative values.
         public static string LandscapeOrPortrait(int width, int height)
         {
-            return default;
+            if (width > height)
+            {
+                return "Landscape";
+            }
+            else if (height > width)
+            {
+                return "Portrait";
+            }
+            else
+                return "Square";
+
```

Comment:
Try to limit the use of return statements to at most, 2 in a method. 3 is a stretch.

Having too many can be confusing for other people who read your code in the future who can can unsure whether or not something is being run. Return statements act as "breaks" in code where literally everything after won't run.  

You can refactor the code above to the following, 

```
var result = "Square";

if (width > height)
{
    result = "Landscape";
}
else if (height > width)
{
    result = "Portrait";
}

return result;
```

This would reduce the output to 1 return statement. This doesn't mean ALWAYS only have 1 return statement however.

Read here for more info: 
https://stackoverflow.com/questions/36707/should-a-function-have-only-one-return-statement#:~:text=Structured%20programming%20says%20you%20should,functions%20with%20multiple%20return%20statements.

(I'm linking stackoverflow to demonstrate what the programming community feels, so you get a feel of a larger audience than just me).